clients/python/sysadmin/ConfigTemplateRenderer.py

```diff
@@ -1,21 +1,11 @@
 import json
 import os
-import pkgutil
 
 from jinja2 import Environment
 from jinja2 import FileSystemBytecodeCache
 from jinja2 import FileSystemLoader
 
-
-class ThunkHelpers(object):
-    pass
-
-
-if len(filter(lambda info: info[1] == 'sysadmin_helpers', pkgutil.iter_modules())):
-    from sysadmin_helpers import helpers
-else:
-    print "Could not find sysadmin helpers, thunking..."
-    helpers = ThunkHelpers
+from sysadmin import helpers
```

Comment:
The real value of the old code here was to not force this client to provide any helpers, but still allow the user of this client to slot in their own helpers as their templates require. I'll agree the `print` statement was annoying, but I'd prefer just removing that to doing this